content/developer/reference/module.rst

```diff
@@ -21,7 +21,7 @@ dictionary, where each key specifies module metadatum.
     {
         'name': "A Module",
         'version': '1.0',
-        'depends': ['base'],
+        'depends': ['mail'],
```

Comment:
Specifying `base` as depends is not wrong, it's even useful (cf. other comment).
Furthermore, I'd prefer specifying a 'dummy' depends in an example than a true depends that would have an effective impact if they don't modify the line.